packages/nginx/README.md

```diff
@@ -41,7 +41,7 @@ Example of a path rule for the default virtual host:
 config location 'ns_location1'
 	option uci_server '_lan'
 	option proxy_pass 'https://192.168.100.234'
-	option description 'Reverse proxy with path'
+	option uci_description 'Reverse proxy with path'
```

Comment:
Is this really needed?
If the change is required, we should also change:
- https://github.com/NethServer/nethsecurity/blob/main/packages/ns-migration/files/scripts/reverse_proxy
- https://github.com/NethServer/nethserver-firewall-migration/blob/master/root/usr/share/nethserver-firewall-migration/reverse_proxy

Reply:
Not required, but heavily suggested. The nginx config generator excludes any fields from the final configuration with `uci_` prefix, so it's just standardizing the behaviour

Reply:
So let's keep the change, but we must fix it also in both migration packages

Reply:
K, will open a PR over there and update `ns-migration` here